test/e2e_kubeadm/util.go

```diff
@@ -0,0 +1,125 @@
+/*
```

Comment:
`util.go` file can maybe be to generic. I have seen it in many project becoming a place holder to everything.. :grin: 
Maybe we should think about having more specific name/files helper even if they are smaller at begin?

Reply:
I agree `util.go` leads to bad practices.
However, the scope of this project is currently limited, and I don't see a real risk that this will get out of control in the foreseeable future.

TBH, In the long term those functions should be moved in the Kubernetes e2e framework because none of them is kubeadm specific, but probably this is something to be discussed in the testing common workgroup (out of the scope of this PR)

Reply:
ok